### DISCUSSION TITLE
How to make a notification ring and stop when the notification is cleared? (Stop a notification script on click)

### DISCUSSION THREAD

**Discussion by DeepReef11** [dunst-project/dunst#1245] [Q&A]
I would like to be able to kill this script when I remove the notification. Is there a way?

```
#!/usr/bin/env sh

while true
do
    paplay "sound.ogg"
    sleep 1
done
```

**Comment by omnigenous** [comment]
Trying to do same thing, did you ever figured this out?

**Reply by omnigenous** [reply]
@bynect Hey, I'm still trying to figure out if this is possible with dunst, could you give your input?

**Reply by omnigenous** [reply]
Oh, you've already replied to similar discussion. I guess there is no solution for this.

https://github.com/dunst-project/dunst/discussions/1382#discussioncomment-10824158
